src/parser/expression.js

```diff
@@ -316,7 +316,7 @@ pp.parseSubscripts = function (base, startPos, startLoc, noCalls) {
     } else if (this.match(tt.backQuote)) {
       let node = this.startNodeAt(startPos, startLoc);
       node.tag = base;
-      node.quasi = this.parseTemplate();
+      node.quasi = this.parseTemplate(true);
```

Comment:
Do you mean with the code duplicated, or with both just calling some `parseTaggedOrUntaggedTemplate(isTagged)`?

Either way, if the concern is just passing a raw boolean, how about instead
```js
const isTagged = true;
node.quasi = this.parseTemplate(isTagged);
```